applications/nrf_desktop/configuration/nrf52840dk_nrf52840/app_ZDebugMCUBootSMP.conf

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 ################################################################################
+# MCUBootQSPI version
+# Configuration that enables MCUmgr SMP over BLE protocol
```

Comment:
with DFU support

Reply:
We enable image management commands for that purpose

Reply:
ok